readme.md

```diff
@@ -26,6 +32,14 @@ To install Administrator as a Composer package to be used with Laravel 4, simply
 )
 ```
 
+### Laravel 5
+
+Then publish the config file with `php artisan vendor:publish`. This will add the file `app/config/administrator.php`. This [config file](http://administrator.frozennode.com/docs/configuration) is the primary way you interact with Administrator.
+
+Next to that all the assets, views and translation files will be published too.
+
+### Laravel 4
+
```

Comment:
I think it is also necessary to tell the users to manually include the following middlewares in their `app/Http/Kernel.php` file inside the `$routeMiddleware` array:

```php
    protected $routeMiddleware = [ 
        ...,       
        'post_validate' => 'Frozennode\Administrator\Http\Middleware\PostValidate',
        'validate_admin' => 'Frozennode\Administrator\Http\Middleware\ValidateAdmin',
        'validate_model' => 'Frozennode\Administrator\Http\Middleware\ValidateModel',
        'validate_settings' => 'Frozennode\Administrator\Http\Middleware\ValidateSettings'
    ];

```